src/components/MaterialFilter.tsx

```diff
@@ -137,27 +143,67 @@ const MaterialFilter: React.FC = () => {
               </label>
             </>
           ))}
-        {/* <label>
+        <label>
           Weight
-          <input className="filter-select" placeholder="Weight" id="weight"></input>
-        </label> */}
+          <input
+            type="number"
+            className="filter-select"
+            id="weight"
+            placeholder="Weight"
```

Comment:
Again, probably best to remove placeholder text.

Reply:
Removed `placeholder = "weight"`